argoverse/utils/camera_stats.py

```diff
@@ -25,7 +25,12 @@
     "ring_side_right",
 ]
 
-STEREO_CAMERA_LIST = ["stereo_front_left", "stereo_front_right"]
+STEREO_CAMERA_LIST = [
+    "stereo_front_left",
+    "stereo_front_right",
+    "stereo_front_left_rect",
```

Comment:
is this new rectified imagery we are releasing, in new zip files?

We do use this variable in some places, and changing like this without a new dataset release would break some of our other demo scripts, so I might prefer to make a new variable

`RECTIFIED_STEREO_CAMERA_LIST = [ blah, blah ] `

Reply:
Agreed. I've changed this and also I created a new function ```load_stereo_calib``` in the ```calibration.py``` so I don't need to mess up with any other demos and tests.

Reply:
Just to clarify -- are we releasing new rectified imagery?

Reply:
Yes, we released rectified imagery.

Reply:
Very cool